### ISSUE TITLE
Mob Penetration Stat Does Not Seem To Work

### ISSUE THREAD

**Issue by Boolyman** [Lycanite/LycanitesMobs#410]
I'm not seeing results from changing the penetration values on mobs.  The way I understand it from your sight, if the mob's damage is set to 10, and the mob's penetration is set to 4.... at least  4 damage from that mob will ignore armor.  Which means the player should always take at least 2 hearts of damage, no?  I mean, unless they have some sort of potion or bauble effect.  

Im not seeing this behaviour.  Im setting some mobs to 6 or 8 penetration, and still only seeing about 1 heart of damage being done.

**Comment by Lycanite** [comment]
Hmm I'll have to look into this one. If you use the command '/lm debug Overlay true' it will turn on a stats overlay when look at a mob close up, can you use this to confirm that their pierce stat is increasing?

**Comment by Boolyman** [comment]
I'm getting an "Invalid Argument" error when trying to use that command.

**Comment by Boolyman** [comment]
twitch.tv/Deadpine is streaming the mod this week so I'd like to give him a good difficult playthrough if you happen to get and ideas about how I can fix this

**Comment by Boolyman** [comment]
I just tested the amor pierce in a vanilla world and it is working there.  So I am going to do some heavy testing to see what mod is causing the conflict.  Will get back to you.

**Comment by Lycanite** [comment]
Yeah managing multiple mods and getting packs right can be a massive pain. Just throw in any issues you find with Lycanites Mobs though and I'm happy to help!
